code/game/objects/structures/grille.dm

```diff
@@ -129,10 +129,10 @@
 		to_chat(user, span_notice("You place the [WD] on [src]."))
 		WD.update_icon()
 
-/obj/structure/grille/fire_act(exposed_temperature, exposed_volume)
-	if(obj_integrity > integrity_failure && exposed_temperature > T0C + 1500)
-		take_damage(1, BURN, FIRE)
-	return ..()
+/obj/structure/grille/fire_act(burn_level)
```

Comment:
this logic here is weird, and is not the same as before

Reply:
other than the temp check, the rest is the same no? The parent call did nothing

Reply:
The temp check's the only difference, but prior to this it always was false so grilles (and windows) never took damage.